labs.yml

```
@@ -185,3 +185,19 @@ commands:
     description: |
       Trigger the migrate-tables workflow and, optionally, migrate-external-hiveserde-tables-in-place-experimental 
       workflow.
+
+  - name: migrate-dbsql-dashboard
```

```suggestion
  - name: migrate-dbsql-dashboards
```

if it's migrate-all, then make it plural